functions.php

```diff
@@ -337,3 +337,25 @@ function twentysixteen_hex2rgb( $color ) {
  * Customizer additions.
  */
 require get_template_directory() . '/inc/customizer.php';
+
+/*
+ * Add image size to enhance responsive image functionality
+ *
+ * @link https://developer.wordpress.org/reference/functions/add_image_size/
+ */
+add_image_size( 'max-content', 600, 600 );
```

Comment:
I'm not familiar with implementing this so let me ask some questions.
 
Should this `add_image_size()` be inside `twentysixteen_setup()`? 

An image in the content can be 840px wide if it's in pages or if it's full-size image in posts. (See screenshot), but does it matter?

![screen shot 2015-10-08 at 06 10 24](https://cloud.githubusercontent.com/assets/908665/10358212/5e1d65ce-6d83-11e5-93c5-d52af256ac23.png)

Do we need to specify the height?